conftest.py

```diff
@@ -101,28 +101,16 @@ def skip_reset(request) -> bool:
 
 @pytest.fixture(scope="session")
 def reset_environment(reset_endpoint, basic_auth, skip_reset):
-    if skip_reset:
```

Comment:
Wondering about this change, the idea was for this to return a function so it can be used in tests, for example here:

https://github.com/NHSDigital/manage-vaccinations-in-schools-testing/blob/36c6212a91a1219a815d429f0bd545c532f7a99e/tests/test_04_school_moves.py#L8